package.json

```diff
@@ -22,59 +22,58 @@
   },
   "private": false,
   "dependencies": {
-    "@angular-devkit/core": "^8.3.29",
-    "@angular-devkit/schematics": "^8.3.29",
-    "@angular/animations": "~8.2.14",
-    "@angular/cdk": "^8.2.1",
-    "@angular/common": "~8.2.14",
-    "@angular/compiler": "~8.2.14",
-    "@angular/core": "~8.2.14",
-    "@angular/forms": "~8.2.14",
-    "@angular/material": "^8.2.1",
-    "@angular/platform-browser": "~8.2.14",
-    "@angular/platform-browser-dynamic": "~8.2.14",
-    "@angular/router": "~8.2.14",
-    "ngx-mat-select-search": "^2.2.0",
+    "@angular-devkit/core": "~10.0.5",
+    "@angular-devkit/schematics": "~10.0.5",
+    "@angular/animations": "~10.0.5",
+    "@angular/cdk": "^10.1.0",
+    "@angular/common": "~10.0.5",
+    "@angular/compiler": "~10.0.5",
+    "@angular/core": "~10.0.5",
+    "@angular/forms": "~10.0.5",
+    "@angular/material": "^10.1.0",
+    "@angular/platform-browser": "~10.0.5",
+    "@angular/platform-browser-dynamic": "~10.0.5",
+    "@angular/router": "~10.0.5",
+    "ngx-mat-select-search": "^3.0.1",
     "nouislider": "^14.0.2",
     "rxjs": "~6.5.5",
-    "tslib": "^1.13.0",
-    "zone.js": "~0.10.3"
+    "tslib": "^2.0.0",
+    "zone.js": "~0.10.2"
```

Comment:
zone.js  downgraded? 0.10.3 to 0.10.2)

Reply:
This results in the same version because of the ~